site/content/docs/main/plugin-release-instructions.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Releasing Velero plugins
+layout: docs
+toc: "true"
+---
+
+# Releasing Velero plugins
```

Comment:
Also needs notes on README compatibility matrix and updated `velero install` instructions.